core-web/apps/dotcms-ui/src/app/view/components/dot-content-compare/store/dot-content-compare.store.spec.ts

```diff
@@ -10,6 +10,72 @@ import {
 } from '@dotcms/data-access';
 import { DotcmsConfigService, LoginService } from '@dotcms/dotcms-js';
 
+const generateRandomString = function (length) {
+    // length is the number of characters or words to generate
+    // type is either 'char' or 'word'
+    let result = '';
+
+    // generate a random string of words
```

Comment:
Here it feels like you can simplify just for example 

```
const generateRandomWords = function (length) {
    const words = ['lorem', 'ipsum', 'dolor', 'sit', 'amet', 'consectetur', 'adipiscing', 'elit'];
    
    return Array.from({ length }, () => words[Math.floor(Math.random() * words.length)]).join(' ');
};
```

Reply:
Done here https://github.com/dotCMS/core/pull/27663/commits/c0a68f0fb5b8636d3eb19847b88ef357ed666687, thanks for your feed back!